### PR TITLE
Uniswap-forks - Sushiswap and Trader Joe. Fixed contract calls to include `try_`.

### PR DESCRIPTION
**Pull request by steegecs**

### CODE REVIEW

File: subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml [hunk 1]
```diff
@@ -4,11 +4,11 @@ schema:
 dataSources:
   - kind: ethereum/contract
     name: LendingPoolAddressesProvider
-    network: {{ network }}
+    network: { { network } }
```
Comment (this-username-is-taken) [inline subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml:7]: you might want to revert this change, assuming it's done by prettier

File: subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleReward.ts [hunk 2]
```diff
@@ -62,26 +66,40 @@ export function handleReward(
...
+
+  // Get the total allocation for all pools
+  let getTotalAlloc = poolContract.try_totalAllocPoint();
+  if (!getTotalAlloc.reverted) {
+    masterChef.totalAllocPoint = getTotalAlloc.value;
+  }
 
   // Address where allocation is moved to over time to reduce inflation
-  let masterPoolAllocPID45 = poolContract.poolInfo(BigInt.fromI32(45)).value1;
+  let getPoolInfo45 = poolContract.try_poolInfo(BigInt.fromI32(45));
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleReward.ts:91]: is this a burner address? might worth adding a bit more comments to explain

File: subgraphs/yearn-v2/src/common/initializers.ts [hunk 3]
```diff
@@ -1,330 +1,334 @@
...
-    );
-
-    vault.fees = [managementFeeId, performanceFeeId];
-
-    vault.save();
-  }
-
-  return vault;
-}
+import {
```
Comment (this-username-is-taken) [inline subgraphs/yearn-v2/src/common/initializers.ts:1]: you should probably revert changes to yearn

File: subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml (hunk 1 above)
Comment (steegecs) [reply subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml:7]: Hmm - Yeah. This might be an issue. This was triggered because I merged with the master branch. Ideally this wouldn't trigger prettier I'd think. Especially since it could affect deployments.

File: subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleReward.ts (hunk 2 above)
Comment (steegecs) [reply subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleReward.ts:91]: Yeah - this is a burner address. I'll put more comments in.

File: subgraphs/yearn-v2/src/common/initializers.ts (hunk 3 above)
Comment (steegecs) [reply subgraphs/yearn-v2/src/common/initializers.ts:1]: Sure -  I will revert these.

File: subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml (hunk 1 above)
Comment (steegecs) [reply subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml:7]: Will want to rebase on master instead of merge.
